sequencer/src/persistence/fs.rs

```diff
@@ -1241,6 +1245,34 @@ impl SequencerPersistence for Persistence {
         Ok(())
     }
 
+    async fn store_drb_input(&self, drb_input: DrbInput) -> anyhow::Result<()> {
+        let inner = self.inner.write().await;
+        let dir_path = inner.drb_dir_path();
+
+        fs::create_dir_all(dir_path.clone()).context("failed to create drb dir")?;
+
+        let drb_input_bytes =
+            bincode::serialize(&drb_input).context("failed to serialize drb_input")?;
+
+        let file_path = dir_path
+            .join(drb_input.epoch.to_string())
+            .with_extension("txt");
```

Comment:
I know we use txt extension at some other places as well, but think the extension should be bin?